resources/application-connector/charts/application-broker/templates/cluster-role.yaml

```diff
@@ -19,7 +19,8 @@ rules:
   verbs: ["get", "create", "delete", "update"]
 - apiGroups: [""]
   resources: ["namespaces"]
-  verbs: ["patch", "list", "watch"]
+  # write access is required because namespaces need to be annotated in order to create a knative broker
+  verbs: ["patch", "list", "watch", "get", "update"]
```

Comment:
I know it was already there before your change, but we _may_ be able to remove `patch`.

Reply:
Sure, no problem. Works without watch. Minimum permissions are always good 👍

Reply:
watch or Patch?

Reply:
Fixed